### PR TITLE
Brand should have max width of 80rem

### LINKED ISSUE #645: Brand should have max width 80rem
**Describe the bug**
Following https://github.com/bbc/psammead/pull/480 the brand is no longer limited to 1290px, whoops

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://bbc.github.io/psammead/iframe.html?id=components-brand--without-brand-link
2. See brand isnt limited to 1280px

**Expected behaviour**
The brand SVG is inside a 1280px wide wrap, to avoid it going all the way to the left of the screen at widths over 1280px

**Screenshots**
image

**Additional context**
No refinement needed, this is a unintended bug

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #645 

**Overall change:** _Update brand to have a max-width of 80rem._

**Code changes:**

- _Update brand to have max-width of 80rem._
- _Update snapshots._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [x] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-brand/src/index.jsx [hunk 1]
```diff
@@ -19,6 +22,11 @@ const PADDING_AROUND_SVG_BELOW_600PX = 32;
 const conditionallyRenderHeight = (svgHeight, padding) =>
   svgHeight ? `height: ${(svgHeight + padding) / 16}rem` : '';
 
+const SvgWrapper = styled.div`
+  max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX};
```
Comment (dr3) [inline packages/components/psammead-brand/src/index.jsx]: Even though [they are the same value](https://github.com/bbc/psammead/blob/7cfe6c0535ed6f95e98ee4fd7eeefc55b212e714/packages/utilities/gel-foundations/src/breakpoints.js#L29-L31) 🙈 Can you use `GEL_GROUP_5_SCREEN_WIDTH_MIN` here instead please :P Its what we used before :)

File: packages/components/psammead-brand/src/index.jsx [hunk 2]
```diff
@@ -100,18 +108,20 @@ const StyledBrand = ({
   <Fragment>
```
Comment (dr3) [inline packages/components/psammead-brand/src/index.jsx]: Also, this doesnt work when the svg is a link weirdly :P May need to move the wrapper up a bit

**Review by j-pendlebury — CHANGES_REQUESTED** [review]
needs a package-lock update, but otherwise LGTM 👍

**Review by j-pendlebury — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by david-boydell** [comment]
Looks good, this is ready for merge!

**Pull request merged**